Library/Homebrew/test/command_options_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "command_options"
+
+describe Homebrew::CommandOptions do
+  describe "Tests for CommandOptions() class: " do
+    it "checks correctness of initialize() method" do
```

Comment:
These messages should describe the test subject rather the test itself — something like `it "initializes correctly"`.